### PR TITLE
Add autoscaling to free tier

### PR DESCRIPTION
**Pull request by bgrenon**
Preview:
- https://neon-next-git-bgrenon-autoscaling-free-tier-neondatabase.vercel.app/docs/introduction/plans#free-tier
- https://neon-next-git-bgrenon-autoscaling-free-tier-neondatabase.vercel.app/docs/introduction/extra-usage
- https://neon-next-git-bgrenon-autoscaling-free-tier-neondatabase.vercel.app/docs/get-started-with-neon/production-checklist#select-the-right-compute-size
- https://neon-next-git-bgrenon-autoscaling-free-tier-neondatabase.vercel.app/docs/ai/ai-scale-with-neon#autoscaling

### CODE REVIEW

File: content/docs/introduction/monitor-usage.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ The peak usage triangle indicates the highest usage level reached for that metri
...
 
   <Admonition type="note">
-  On the Free Tier, the [default branch](/docs/reference/glossary#default-branch) compute is a 0.25 vCPU compute that is always available, so allowances do not apply to your default branch. You can run your 0.25 vCPU compute on the Free Tier 24/7. Only branch computes on the Free Tier have an allowance, which is the 5 compute hour/month allowance that Free Tier users see on the **Billing** page. On the Free Tier, this is actually 20 hours of usage because the compute size on the Free Tier is 0.25 vCPU. You cannot increase the compute size on the Free Tier.
+  On the Free Tier, [default branch](/docs/reference/glossary#default-branch) compute starts at 0.25 vCPU and can run 24/7. Autoscaling up to 2 vCPU with 8 GB RAM is available for extra performance during peak times; note that autoscaling can affect 24/7 availability depending on your usage patterns. Your account includes up to 191.9 compute hours per month, with 5 compute hours/month allocated to non-default branches.
```
Comment (danieltprice) [inline content/docs/introduction/monitor-usage.md]:
```suggestion
  On the Free Tier, [default branch](/docs/reference/glossary#default-branch) compute starts at 0.25 vCPU and can run 24/7. Autoscaling up to 2 vCPU with 8 GB RAM is available for extra performance during peak times; note that autoscaling can affect 24/7 availability depending on your usage patterns. The Free Tier includes up to 191.9 compute hours per month, with up to 5 compute hours/month available to non-default branches.
```

File: content/docs/introduction/plans.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ The Free Tier includes the following usage allowances:
...
+On the Free Tier, your default branch compute can run 24/7 at 0.25 CU. If you enable autoscaling, your compute can scale up to 2 vCPU with 8 GB RAM, providing additional resources to meet peak demand. Note that enabling autoscaling may affect 24/7 availability depending on your usage patterns. Your account includes 191.9 compute hours per month, with 5 compute hours per month allocated to non-default branches. If you go over either allowance, that compute is suspended until the allowance resets at the beginning of the month. For example, if you signed up for the Free Tier in January, your compute allowance resets on February 1st.
```
Comment (danieltprice) [inline content/docs/introduction/plans.md:71]:
```suggestion
On the Free Tier, your default branch compute can run 24/7 at 0.25 CU. If you enable autoscaling, your compute can scale up to 2 CU, providing additional resources to meet peak demand. Note that enabling autoscaling may affect 24/7 availability depending on your usage patterns. The Free Tier includes 191.9 compute hours per month, with up to 5 of those compute hours per month available to non-default branches. If you go over the 5 compute hours allowance, non-default branch computes are suspended until the allowance resets at the beginning of the month. If you go over the 191.9 compute hour allowance, all computes are suspended until the beginning of the month. For example, if you signed up for the Free Tier in January, your compute allowance resets on February 1st.
```

File: content/docs/ai/ai-scale-with-neon.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ In Neon, scaling vertically is a matter of selecting the desired compute size. N
 | 9                  | 9    | 36 GB | 604 MB               |
 | 10                 | 10   | 40 GB | 671 MB               |
 
-See [Edit a compute](/docs/manage/endpoints#edit-a-compute) to learn how to configure your compute size. Available compute sizes differ according to your Neon plan. The Neon Free Tier supports computes with 0.25 vCPU and 1 GB RAM. The Launch plan offers compute sizes up to 4 vCPU with 16 GB RAM. Larger computes are available on the Scale plan. See [Neon plans](/docs/introduction/plans).
+See [Edit a compute](/docs/manage/endpoints#edit-a-compute) to learn how to configure your compute size. Available compute sizes differ according to your Neon plan. The Neon Free Tier supports computes starting at 0.25 vCPU with 1 GB RAM, up to 2 vCPU with 8 GB RAM with autoscaling enabled. The Launch plan offers compute sizes up to 4 vCPU with 16 GB RAM. Larger computes are available on the Scale plan. See [Neon plans](/docs/introduction/plans).
```
Comment (danieltprice) [inline content/docs/ai/ai-scale-with-neon.md:27]:
```suggestion
See [Edit a compute](/docs/manage/endpoints#edit-a-compute) to learn how to configure your compute size. Available compute sizes differ according to your Neon plan. The Neon Free Tier supports computes starting at 0.25 CU, up to 2 CU with 8 GB RAM with autoscaling enabled. The Launch plan offers compute sizes up to 4 CU. Larger computes are available on the Scale plan. See [Neon plans](/docs/introduction/plans).
```

**Comment by danieltprice** [comment]
We're using CU and VCPU interchangeably. We should probably read through these topics to make sure we're using those terms in a way that not's too confusing. For example, avoid switching between the two in the same paragraph or page. If necessary, you could link "CU" to the glossary definition:
https://neon.tech/docs/reference/glossary#compute-unit-cu

**Comment by bgrenon** [comment]
> We're using CU and VCPU interchangeably. We should probably read through these topics to make sure we're using those terms in a way that not's too confusing. For example, avoid switching between the two in the same paragraph or page. If necessary, you could link "CU" to the glossary definition: https://neon.tech/docs/reference/glossary#compute-unit-cu

Let's see if we can go with these variants:
- If the page introduces/is focused on the concept of CU (because we're talking generally about available sizes), stick to CU with no mention of RAM.
- If we need to mix CU with RAM, then use parentheses. Example: `2 CU (2 vCPU with 8 GB RAM)`
- If we are on a page without using CU (because we're talking about precise sizing), use vCPU + RAM. Example: `Increase your size to 2 vCPU with 8 GB RAM`

File: content/docs/introduction/plans.md [hunk 4]
```diff
@@ -35,14 +35,14 @@ Neon's Free Tier plan is best for hobby projects, prototypes, and learning Neon.
...
+| **Projects**               | 1 Neon project                                                                                                                                                                                 |
+| **Branches**               | 10 branches                                                                                                                                                                                    |
+| **Databases**              | Unlimited                                                                                                                                                                                      |
+| **Storage**                | 0.5 GiB                                                                                                                                                                                        |
+| **Compute**                | 24/7 database at 0.25 CU. Autoscaling up to 2 vCPU with 8 GB RAM available. Your account includes 191.9 compute hours per month, with 5 compute hours/month allocated to non-default branches. |
```
[the remark this answers is not in the document]
Comment (bgrenon) [reply content/docs/introduction/plans.md]: Fixed.

File: content/docs/introduction/monitor-usage.md (hunk 1 above)
Comment (bgrenon) [reply content/docs/introduction/monitor-usage.md]: Using full "vCPU + RAM" convention now, avoiding mixing CU and vCPU altogether:

`On the Free Tier, [default branch](/docs/reference/glossary#default-branch) compute starts at 0.25 vCPU with 1 GB RAM and can run 24/7. Autoscaling up to 2 vCPU with 8 GB RAM is available for extra performance during peak times; note that autoscaling can affect 24/7 availability depending on your usage patterns. Your account includes 191.9 compute hours per month, with up to 5 of those hours available to non-default branches.`

**Review by danieltprice — APPROVED** [review]

**Review by bvanvugt — APPROVED** [review]

**Pull request merged**